packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -410,6 +410,41 @@ def _get_bundle_list(dcos_api_session):
     return bundles
 
 
+@retrying.retry(wait_fixed=2000, stop_max_delay=120000,
+                retry_on_result=lambda x: x is False)
+def wait_for_diagnostics_job(dcos_api_session, last_datapoint):
+    response = check_json(dcos_api_session.health.get('report/diagnostics/status/all'))
+    # find if the job is still running
+    job_running = False
+    percent_done = 0
+    for _, attributes in response.items():
```

Comment:
FYI if you are just throwing away the keys you could use `response.values()`

Reply:
Yep. Again, this was part of the stuff I copy-paste-refactored and I didn't bother going through and refining it.